doc/conf.py

```diff
@@ -201,7 +201,12 @@ def _custom_edit_url(github_user, github_repo, github_version, doc_path, file_na
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "_static/vispy-teaser-short.png"
+
+html_static_path = ["_static"]
+html_theme_options = {
+    "light_logo": "logo-light-mode.png",
+    "dark_logo": "logo-dark-mode.png",
+}
```

Comment:
The documentation you linked to for implementing this was for the "Furo" sphinx theme which we aren't using. We are using the pydata sphinx theme which has documentation for doing light/dark logos here:

https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/branding.html#different-logos-for-light-and-dark-mode

You'll need to change this to match. Additionally you are overwriting the `html_theme_options` dictionary. You'll need to move these change to the existing `html_theme_options` dictionary (starting on line 185). Similarly, the `html_static_path` is already defined a couple lines below your changes.